nav-fim-dokumentproduksjon-v2-tjenestespesifikasjon/pom.xml

```diff
@@ -0,0 +1,111 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<groupId>no.nav.tjenestespesifikasjoner</groupId>
+		<artifactId>tjenestespesifikasjoner</artifactId>
+		<version>0-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>nav-fim-dokumentproduksjon-v2-tjenestespesifikasjon</artifactId>
+	<version>0-SNAPSHOT</version>
+
+	<properties>
+		<wsdl.directory>src/main/resources/wsdl/no/nav/tjeneste/virksomhet/dokumentproduksjon/v2</wsdl.directory>
+	</properties>
+
+	<build>
+		<plugins>
```

Comment:
Husk også å legge inn maven-javadoc-plugin og maven-sources-plugin, for å generere sources og javadoc - påkrevd for Maven Central. (Se de andre modulene i dette prosjektet, hvordan de gjør det.)

Reply:
Takk for tips. Satt opp iht standard her.
På sikt bør vel tjenesteleverandørene publisere og man trenger en versjonshåndtering

Reply:
Stemmer. Det er noen som allerede publiserer selv. Så dette monorepoet er et slags catch-all for de som ikke gjør det.